versioned_docs/version-3.x.xx/platform/android/adding-sharpsell-sdk-android.md

```diff
@@ -84,9 +67,43 @@ dependencies {
     implementation 'com.google.firebase:firebase-messaging-ktx'
     implementation 'com.google.firebase:firebase-crashlytics-ktx'
 
-    implementation 'com.enparadigm.sharpsell:sdk:$sdkVersion'
+    implementation ("com.enparadigm.sharpsell:sdk:$sdkVersion"){
+        exclude group: 'io.flutter', module: 'flutter_embedding_debug'
+        exclude group: 'io.flutter', module: 'flutter_embedding_profile'
+    }
 }
 ```
+
+3. Add the following lines to the app-level `proguard-rules.pro` file if you have one.
```

Comment:
instead we should say, if you are using proguard rules in your app please add the below rules to you pro guard-rules.pro file. because it is not a mandatory step.